htlcswitch/link.go

```diff
@@ -346,6 +351,8 @@ type channelLink struct {
 	// by the HTLC switch.
 	downstream chan *htlcPacket
 
+	downstreamIn chan *downstreamMsg
```

Comment:
came could be more descriptive, e.g `localDownstream`

Reply:
fixed, good suggestion